packages/astro/test/fixtures/component-library/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@test/component-library",
+  "version": "0.0.0",
+  "private": true,
+  "dependencies": {
+    "astro": "workspace:*",
+    "@astrojs/preact": "workspace:*",
+    "@astrojs/react": "workspace:*",
+    "@astrojs/svelte": "workspace:*",
+    "@test/component-library-shared": "workspace:*",
```

Comment:
I'm always a little weary of `workspace:*` as a package test. Is it possible to generate a tarball from `component-library-shared` via `pnpm pack`, and reference that file from here? This would 100% mock an external npm dependency

Reply:
I'm not actually sure if tarballs work properly in vite 🤔 I don't use `pnpm pack` often so I may have it misconfigured, but I tried this locally and Vite blew up with errors failing to resolve anything inside the tarball dependency

`link:../component-library-shared` would work for the tests, but I'm not sure the details of how that's different from `workspace:*`